### PR TITLE
Clarify isConnected usage to clearly demonstrate bool return case

### PR DESCRIPTION
**Pull request by peitschie**
Related to discussion on #1015

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -160,9 +160,14 @@ declare namespace BLECentralPlugin {
...
         stopNotification(device_id: string, service_uuid: string, characteristic_uuid: string): Promise<void>;
 
-        /* Returns a rejected promise if the device is not connected */
+        /* Returns a resolved promise if the device is connected,
+            otherwise returns rejected promise if the device is not connected */
         isConnected(device_id: string): Promise<void>;
 
+        /* Returns a promise that resolves to true if the device is connected,
+            otherwise resolves to false if the device is not connected or an error occurs */
+        isConnected(device_id: string, rejectWhenDisconnected: false): Promise<boolean>;
```
Comment (MaximBelov) [inline types.d.ts:169]: Type of argument rejectWhenDisconnected should be boolean instead of false

**Review by MaximBelov — CHANGES_REQUESTED** [review]

File: types.d.ts (hunk 1 above)
Comment (peitschie) [reply types.d.ts:169]: By deliberately setting it to `false`, I can narrow the return type  to `Promise<boolean>`.

Calling this with `true` is effectively the same as the `resolve` or `reject` overload above.

Comment (MaximBelov) [reply types.d.ts:169]: Yep, sorry
You are right 
I missed the definition without the second argument 
isConnected(device_id: string): Promise<void>;

**Comment by peitschie** [comment]
This has been pushed in v1.7.4

**Pull request merged**